#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Adjust ISO 4217 currency definition for TRY
# DP: Related bugs: Ubuntu 9743
# DP: Dpatch author: Jeff Bailey
# DP: Patch author: Jeff Bailey
# DP: Upstream status: Not submitted
# DP: Status Details: 
# DP: Date: 2005-04-21

PATCHLEVEL=0

if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p$PATCHLEVEL < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p$PATCHLEVEL < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

# append the patch here and adjust the -p? flag in the patch calls.
--- locale/iso-4217.def.old	2005-04-21 06:43:11.881144068 +0000
+++ locale/iso-4217.def	2005-04-21 06:43:31.463482872 +0000
@@ -150,7 +150,7 @@
 DEFINE_INT_CURR("TMM")		/* Turkmenistan Manet  */
 DEFINE_INT_CURR("TND")		/* Tunisian Dinar  */
 DEFINE_INT_CURR("TOP")		/* Tonga Pa'Anga  */
-DEFINE_INT_CURR("TRL")		/* Turkish Lira  */
+DEFINE_INT_CURR("TRY")		/* Turkish Lira  */
 DEFINE_INT_CURR("TTD")		/* Trinidad and Tobago  */
 DEFINE_INT_CURR("TVD")		/* Tuvalu Dollars  */
 DEFINE_INT_CURR("TWD")		/* Taiwan, Province of China Dollar  */
